signer/src/config/mod.rs

```diff
@@ -349,6 +349,8 @@ pub struct SignerConfig {
     /// The number of bitcoin blocks after a DKG start where we attempt to
     /// verify the shares. After this many blocks, we mark the shares as failed.
     pub dkg_verification_window: u16,
+    /// The maximum stacks fee in microSTX that the signer will accept for any stacks transaction.
+    pub stx_fee_max_micro_stx: NonZeroU64,
```

Comment:
I might have named this `stacks_fees_max_ustx` (where maybe we have a corresponding `bitcoin_fees_max_sats` for bitcoin), just because it flows a little nicer 🤷